namespaces/live.cloud-platform.service.justice.gov.uk/hale-platform-dev/resources/irsa.tf

```diff
@@ -13,10 +13,56 @@ module "irsa" {
   # provide an output called `irsa_policy_arn` that can be used.
   role_policy_arns = {
     s3 = module.s3_bucket.irsa_policy_arn
+    s3_lawcom = module.aws_iam_policy.tacticalproducts_lawcom_s3_policy.irsa_policy_arn,
     ecr = module.ecr_credentials.irsa_policy_arn,
     ecr2 = module.ecr_feed_parser.irsa_policy_arn,
   }
 
+  data "aws_iam_policy_document" "tacticalproducts_lawcom_s3_policy" {
+    # Provide list of permissions and target AWS account resources to allow access to
+    statement {
+      actions = [
+        "s3:ListBucket",
+      ]
+      resources = [
+        "arn:aws:s3:::lawcom-prod-storage-11jsxou24uy7q",
+     ]
+    }
+    statement {
+      actions = [
+        "s3:*",
```

Comment:
If the intention of this policy is to permit uploading objects to a remote bucket, the actions list could be smaller.